certbot-apache/certbot_apache/tests/parsernode_test.py

```diff
@@ -13,14 +13,29 @@ class DummyParserNode(interfaces.ParserNode):
     dirty = False
     filepath = None
 
+    def __init__(self, ancestor=None, filepath=str(), dirty=False):
```

Comment:
Since we're adding `__init__` in this PR now too, can we delete the class variables in this file?

Reply:
Yeah, I forgot to remove them previously!